bigtable/metrics_util.go

```diff
@@ -30,6 +30,7 @@ import (
 const (
 	defaultCluster = "unspecified"
```

Comment:
```suggestion
	defaultCluster = "<unspecified>"
```

Reply:
Java has "unspecified": https://github.com/googleapis/java-bigtable/blob/168293937cc7f438a3ec2dee46805aa8e12089c4/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java#L78-L80

Reply:
Updated to <unspecified>